maintainers/maintainer-list.nix

```diff
@@ -12230,6 +12230,11 @@
     github = "joshniemela";
     githubId = 88747315;
   };
+  joshprk = {
+    name = "Joshua Park";
+    github = "joshprk";
+    githubId = 123624726;
+  };
```

Comment:
Split this change out to a separate commit *preceding* the main commit, titled `maintainers: add joshprk`